### PR TITLE
17 add scroll to top option

### LINKED ISSUE #17: Add Scroll To Top option

### PR DESCRIPTION
**Pull request by SychaoChin**

### CODE REVIEW

**Comment by PitouBruh** [comment]
working as intented

**Review by PitouBruh — APPROVED** [review]

File: app/src/main/java/com/example/wallet/MainActivity.kt [hunk 1]
```diff
@@ -26,6 +29,33 @@ class MainActivity : AppCompatActivity() {
         setCurrentFragment(firstFragment)
 
         val bottomNavigationView: BottomNavigationView = findViewById(R.id.bottomNavigationView)
+        bottomNavigationView.setOnItemReselectedListener { item ->
+            when (item.itemId) {
+//                R.id.homeFragment -> firstFragment.view?.findViewById<RecyclerView>(R.id.upcoming_payment_View)?.scrollToPosition(0)
+//                R.id.chartFragment -> thirdFragment.view?.findViewById<RecyclerView>(R.id.upcoming_payment_chart_View)?.scrollToPosition(0)
```
Comment (Sovannra) [inline app/src/main/java/com/example/wallet/MainActivity.kt]: Does this comment code still use? If you don't use it anymore, please remove it.

Comment (SychaoChin) [reply app/src/main/java/com/example/wallet/MainActivity.kt]: Done

File: app/src/main/java/com/example/wallet/MainActivity.kt [hunk 2]
```diff
@@ -26,6 +29,28 @@ class MainActivity : AppCompatActivity() {
         setCurrentFragment(firstFragment)
 
         val bottomNavigationView: BottomNavigationView = findViewById(R.id.bottomNavigationView)
+        bottomNavigationView.setOnItemReselectedListener { item ->
+            when (item.itemId) {
+                R.id.homeFragment -> {
+                if(firstFragment.isAdded() && firstFragment.isVisible()){
```
Comment (Sovannra) [inline app/src/main/java/com/example/wallet/MainActivity.kt]: Please define variable name to be specific. 
Example: 'firstFragment' -> 'homeFragment'.
'thirdFragment' -> 'chartFragment'.

**Pull request merged**